doc/user_guide/impala_user_guide.md

```diff
@@ -135,6 +182,17 @@ CREATE VIRTUAL SCHEMA <virtual schema name>
    SCHEMA_NAME     = '<schema name>';
 ```
 
+### Enabling Logging
+
+You can add `LogLevel` and `LogPath` parameters to the JDBC URL (the `TO` part of connection object) to enable Impala JDBC driver and connection logs.
+
+For example:
+
+```
+TO 'jdbc:impala://<Impala host>:<port>;AuthMech=1;KrbAuthType=1;KrbRealm=EXAMPLE.COM;KrbHostFQDN=_HOST;KrbServiceName=impala;transportMode=http;httpPath=cliservice;LogLevel=6;LogPath=/tmp/'
```

Comment:
Could you add a reference to a place where the meaning and available values for LogLevel are explained?